package.json

```diff
@@ -68,18 +68,18 @@
     "atlassian-jwt": "^2.0.0",
     "aws-sdk": "^2.999.0",
     "axios": "^0.25.0",
-    "body-parser": "^1.18.3",
+    "body-parser": "^1.19.1",
```

Comment:
I wonder how does the library version bump help the folder structure?

Reply:
I was getting eslint resolver issues and there was an issue with the express-validator that I needed to upgrade to the latest to use it appropriately for the new routing. While I was here, I saw that there were a few things related to routing/express that needed to upgrade and figured if we're testing all these routes, it would make sense to update these packages instead of updating them and retesting all the routes anyways.